.browserslistrc

```diff
@@ -0,0 +1,4 @@
+# Browsers that we support
+
+last 1 version
+> 3%
```

Comment:
Hey Josh, I know this just got merged, but I think this browserslist might support more browsers than we want to support in this repo.

> covers the latest version of modern browsers with significant market share.

[`last 1 version`](https://browsersl.ist/?q=last+1+version) includes outdated browsers like IE11 and IE Mobile and Blackberry Browser. This will result in more transpilation than I think is necessary.

Can we limit this to evergreen browsers? If we decide that our main cloudfour site needs to support more browsers than this repo supports, we can add more transpilation there.

How about:

```
>2%
last 1 safari version
last 1 ios version
last 1 firefox version
last 1 chromeandroid version
```

[Preview](https://browsersl.ist/?q=%3E2%25%2C+last+1+safari+version%2C+last+1+ios+version%2C+last+1+firefox+version%2C+last+1+chromeandroid+version)

cc @tylersticka

Reply:
I'm willing to submit a PR if that is helpful

Reply:
@calebeby Go for it! Or create an issue if time is a factor, either way.

Reply:
✅ https://github.com/cloudfour/cloudfour.com-patterns/pull/658